modules/end-user-guide/partials/proc_configuring-a-workspace-to-use-an-intellij-based-ide.adoc

```diff
@@ -0,0 +1,7 @@
+[id="configuring-a-workspace-to-use-an-intellij-based-ide_{context}"]
+= Configuring a workspace to use an IntelliJ-based IDE
+
+This section describes how to configure a workspace to use an IntelliJ-based IDE.
+
+include::example$snip_{project-context}-configuring-a-workspace-to-use-an-intellij-based-ide.adoc[]
```

Comment:
I might be missing something here, but are you sure you want this as an example? It looks like a solid procedure inside that file. So you might want to consider pasting here, after responding to my review of it above, the complete content of examples/modules/end-user-guide/examples/snip_che-configuring-a-workspace-to-use-an-intellij-based-ide.adoc and deleting that file altogether.

Reply:
There is one more issue with having it as an example rather than the procedure - the example does not appear in the TOC tab like a procedure would: 
![image](https://user-images.githubusercontent.com/78564304/132507699-7a792b67-9168-434f-a267-750d675b25ac.png)

Reply:
This is an example because the downstream version will be quite different (at least for one release or two).

Reply:
As the page is a single procedure, no toc is expected.

Reply:
Rewritten as 3 procedures